src/app_service/service/community/service.nim

```diff
@@ -1148,8 +1149,12 @@ QtObject:
       return
 
     let community = rpcResponseObj{"result"}.toCommunityDto()
-    self.allCommunities[community.id] = community
-    self.events.emit(SIGNAL_COMMUNITY_DATA_IMPORTED, CommunityArgs(community: community))
+    if community.id != "":
+      self.allCommunities[community.id] = community
+      self.events.emit(SIGNAL_COMMUNITY_DATA_IMPORTED, CommunityArgs(community: community))
+    else:
+      self.events.emit(SIGNAL_COMMUNITY_LOAD_DATA_FAILED, CommunityArgs(error: "Couldn't find community info"))
```

Comment:
Why don't we include community id in the signal args? Do we support only one import at a time? What is a timeout for fetching community data?

Reply:
There is an example: https://github.com/status-im/status-desktop/blob/5dcd3dde23a06660011bc3031af78306e80ae5ee/src/app_service/service/stickers/async_tasks.nim#L52-L53

Reply:
Ah, nice! Thank you!!